### DISCUSSION TITLE
mongodb_keyfile_content

### DISCUSSION THREAD

**Discussion by dikshauat** [Sunbird-Ed/Community#174] [General Discussion]
Hi @Shounak, @keshavprasad, please let me know the process of generating the value for mongodb_keyfile_content variable of secrets.yml

Regards

**Comment by beepdot** [comment]
Please see here - https://github.com/project-sunbird/sunbird-devops/blob/master/private_repo/ansible/inventory/dev/Core/secrets.yml#L68-L88
